lib/task_helpers/imports/provision_dialogs.rb

```diff
@@ -0,0 +1,24 @@
+module TaskHelpers
+  class Imports
+    class ProvisionDialogs
+      def import(options = {})
+        return unless options[:source]
+
+        glob = File.file?(options[:source]) ? options[:source] : "#{options[:source]}/*.yaml"
+        Dir.glob(glob) do |fname|
+          $log.send(:info, "Importing Provision Dialog from: #{fname}")
+
+          dialog = YAML.load_file(fname)
+
+          miq_dialog = MiqDialog.where(:name => dialog[:name]).first
```

Comment:
Prefer ```find_by(:name => dialog[:name])``` instead of ```where().first```

Reply:
These are uniq'd by dialog type, does this need to account for that as well?

Reply:
Both the export and import need to account for them being uniq'd by dialog type. On the export side with the way the filename is generated when there are two dialogs of different types have the same name then one will overwrite the other.